linux/user-and-file-management/bash-environment/customizing-bash-via-bashrc.md

```diff
@@ -13,40 +9,40 @@ type: normal
 
 category: must-know
 
-inAlgoPool: false
-
 tags:
   - introduction
 
-parent: what-is-a-shell
-
 ---
 
-# Customizing `bash` via `.bashrc`
+# Personalizing Your Environment
 
 ---
 ## Content
 
-It's possible for each user to customize their `bash` environment.  This is done by editing the `.bashrc` file in a given user's home directory.
+It's possible for each user to customize their `bash` environment. This is done by editing the `.bashrc` file in their home directory.
 
-One of the first things `bash` does when loaded is look for the `.bashrc` file in the current user's home directory.  If the file exists, each line in this file is executed "as if" the user typed the line directly at the prompt.
+One of the first things `bash` does when loaded is look for the `.bashrc` file in the current user's home directory. If the file exists, each line in this file is executed "as if" the user typed the line directly in the terminal.
 
 ### Common Customizations
 
 Common customizations include:
-
-- Changing the default prompt by adding color or additional information
-- Using `alias` to create short-hand expressions for common commands, e.g., making `ll` an alias of `ls -l`.
+- Changing the default terminal by adding color or additional information
+- Using `alias` to create short-hand expressions for common commands
 - Changing system defaults, usually by modifying environment variables such as `PATH` and `EDITOR`.
 
+> **Note:** when researching how to customize your shell, you'll often see reference to both `.bashrc` and `.bash_profile`.  Most systems are set up so that the difference between these two files doesn't matter and you can edit one or the other to the same effect[1].
```

Comment:
I skipped commenting on some of these but I'd use the 💡everywhere instead of "Note:" to provide bit more visual diversity